docs/_docs/03-ug-mutable-dataset.md

```diff
@@ -106,6 +121,32 @@ hs = Hyperspace(spark)
 hs.refreshIndex("empIndex", "incremental")
 ```
 
+#### Refresh Index - Quick Mode
+
+Refresh Quick mode is metadata only. It collects the list of appended and deleted source data files at refresh time, and does not update any index data.
+To leverage the difference of source data files, Hyperspace utilizes [Hybrid Scan](#hybrid-scan) when applying the index, even if Hybrid Scan is disabled.
+It relies on on-the-fly shuffle and merge for appended files and injected filter condition for deleted files.
+Newly processed data will not be persisted. 
```

Comment:
Could you clarify on this? "Newly processed data will not be persisted."

Reply:
~ will not be persisted, thus Shuffle for appended files will be performed for every query execution utilizing the index.

I intended this, but not sure it's necessary here.

Reply:
"Newly processed data" was a bit confusing to me (note that we have appended/deleted source "data" and index "data" in this paragraph). Maybe we can just delete this line?